build-tools/src/testFixtures/groovy/org/elasticsearch/gradle/fixtures/AbstractGradleFuncTest.groovy

```diff
@@ -136,6 +143,35 @@ abstract class AbstractGradleFuncTest extends Specification {
         return jarFile;
     }
 
+    //todo maybe we could refer to ../libs/plugin-api compile it and create a jar?
```

Comment:
any views on this? is this possible?

Reply:
I don't see how this is possible. We run into a chicken egg problem here